code/game/machinery/doors/airlock.dm

```diff
@@ -72,6 +72,7 @@
 	var/hasShocked = 0 //Prevents multiple shocks from happening
 	autoclose = 1
 	var/obj/item/device/doorCharge/charge = null //If applied, causes an explosion upon opening the door
+	var/obj/item/weapon/note //Any papers pinned to the airlock
```

Comment:
>we are missing sprites in other places so we should miss them here

Not an argument. You opened this can of worms by adding such a minor visual feature.